pep-0001.txt

```diff
@@ -457,19 +457,31 @@ if they are never meant to be completed.  E.g. :pep:`1` (this PEP).
 PEP Maintenance
 ---------------
 
-In general, Standards track PEPs are no longer modified after they have
-reached the Final state. Once a PEP has been completed, the Language and
-Standard Library References become the formal documentation of the expected
-behavior.
+In general, PEPs are no longer substantially modified after they have reached
+the Accepted, Final, Rejected or Superseded state. Once resolution is reached,
+a PEP is considered a historical document rather than a living specification.
+Formal documentation of the expected behavior should be maintained elsewhere,
+such as the `Language Reference`_ for core features, the `Library Reference`_
+for standard library modules or the `PyPA Specifications`_ for packaging.
 
 If changes based on implementation experience and user feedback are made to
-Standards track PEPs while in the Accepted or Provisional State, those changes
-should be noted in the PEP, such that the PEP accurately describes the state of
+Standards track PEPs while in the Provisional or (with SC approval) Accepted
```

Comment:
Do we specify anywhere else that changes to Accepted PEPs require re-approval? If so, the `(with SC approval)` can be taken out. If not, please can this be rephrased to not have the parenthetical aside.

Reply:
Not that I could find, but its possible I missed it—this PEP is rather long and its organization sometimes results in things being different places. I was basing this off @gvanrossum 's reply here and what seems to be accepted (heh) practice.

We can do this, if you prefer:

```suggestion
Standards track PEPs while in the Provisional (or with SC approval, Accepted)
```

Off the top of my head, I'm not sure I can think of another straightforward way of avoiding this while still being explicit and unambitious that it only applies to `Accepted` and not `Provisional`, at least without rewriting this whole otherwise unchanged paragraph. Could you maybe suggest something, and/or explain the issue with the parenthetical?

Reply:
> explain the issue with the parenthetical

A general aversion to parenthetical asides in formal writing. Suggestion below, though it is a re-write of the entire paragraph.

A

```rest
While implementing a Standards Track PEP, it may be desirable to alter the
behaviour described in the PEP due to user feedback or implementation 
experience. The PEP should be updated such that it accurately describes
the implementation at the point where it is marked Final. Changes to the
behaviour described in an Accepted PEP must also recieve approval from the 
`Steering Council`_.
```

Reply:
I'm not sure we need to completely re-write this paragraph from the original just to suit one person's personal aversion to parenthetical asides, but what do others think (@JelleZijlstra @hugovk )?

Reply:
I think it's fine as is.